src/app/state/auth/auth.selectors.spec.ts

```diff
@@ -24,7 +24,7 @@ describe('Auth Selectors', () => {
     it('should return false when metamaskChainId is set to Volta chain', () => {
       expect(
         authSelectors.isMetamaskDisabled.projector({
-          metamask: { chainId: '0x12047' },
+          metamask: { chainId: 73799 },
```

Comment:
Do we want to move Ids to enum or variable or config file or something?

Reply:
This might be a good idea to create something. I will think about it, what will be the best here :) 
Is there something you prefer?

Reply:
No, whatever you think is best! A config file maybe that depends on env? As I think chain ID is dependent on that?

Reply:
I created an object for storing EWC and Volta chain ids, reused in all places that i was able to find.